docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
     volumes:
       - db_volume:/pgdata
   contrib:
-    image: vimc/montagu-contrib-portal:${MONTAGU_CONTRIB_PORTAL_VERSION}
+    image: ${VIMC_REGISTRY}/montagu-contrib-portal:${MONTAGU_CONTRIB_PORTAL_VERSION}
```

Comment:
Should this *always* be `vimc` now, as we move to deprecate the old registry?

Reply:
It should be, not sure all the images are getting uploaded there yet - we've got a couple of epics in the current sprint for this we should try to move forward...

Reply:
The portals are on docker hub and in fact no longer exist on the public registry, hence 'vimc' here!